### DISCUSSION TITLE
Aristotelian Definitions and Object Properties

### DISCUSSION THREAD

**Discussion by gregfowlerphd** [JisooSeo/101-WG-Discussions#15] [General]
It’s generally agreed to be best practice in ontology development, at least in BFO circles, for classes to be provided Aristotelian definitions whenever possible. That is, roughly, where A is a parent class of B, the ideal form of the definition provided for B is thought to be:

> An A that Cs

where C is whatever distinguishes instances of B from other instances of A.

However, there appears to be no such standard in place for object property definitions, and I wonder whether there should be. Of course, object properties cannot strictly speaking be provided Aristotelian definitions, but something analogous can be done. Roughly, where R2 is a subproperty of R1, the definition for R2 could be of the form:

> x R2s y iff x R1s y and C(x, y)

where C is a condition that distinguishes pairs between which R2 obtains from other pairs between which R1 obtains.

So, what do y'all think? Would adopting the standard that definitions for object properties, whenever possible, be of this form have benefits similar to the Aristotelian definition standard for classes? Overall, would doing so be a good thing?

(Since I know this repo doesn't get much traffic, let me just tag some people here who might be interested: @JisooSeo, @CarterBeauBenson, @johnbeve, @hectorguzor, @michaelrabenberg, @Regina-Hurley, @mark-jensen.)

**Comment by johnbeve** [comment]
This is similar to the strategy I use when creating object property definitions. There is not always an appropriate parent property, I include domain/range, and it is unlikely you'll in every case find something satisfying C(x,y). For what it's worth, in the formula for classes, it is not assumed there is some C(x), since differentia can be gnarly. 

What I do schematically: 

Relation(x,y) iff Domain(x) & Range(y) & Parent_Relation(x,y) & C

Where the parent property is optional. For example, from a recent paper (observe: no parent property): 

is accredited by =def A relation that holds between a credential granting agency and a quality assurance group when the credential granting agency satisfies accreditation standards set by, and confirmed to have been satisfied by, the quality assurance group.

**Comment by gregfowlerphd** [comment]
@johnbeve:

Thanks for your reply, John. :)

Yes, your schema for object property definitions would satisfy the standard I floated, since I didn't intend it to require the "x R1s y" clause to come first and C(x, y) was intended to be an open formula with x and y as free variables.

In the class definitions case, I was thinking of "C" as essentially standing for an open formula as well (though I didn't include the variable since, in contrast to object property definitions, class definitions don't often contain variables). Given this understanding, would the formula cover cases of gnarly differentia, since open formulas can themselves be pretty gnarly?
